frontend/web/styles/_variables.scss

```diff
@@ -1,9 +1,11 @@
+@import "new/variables-new";
```

Comment:
is using `new` here the best option? Seems like the sort of thing that might get looked back on in a years time with annoyance...

Reply:
It'll eventually replace the whole variables file once the designs are complete - they both have to exist in parallel for a while

Reply:
Each pr should delete from the old one and add to the new until we rename